src/Lykke.Job.OperationsCache/Services/InMemoryCache/InMemoryStorage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Lykke.Job.OperationsCache.Models;
+
+namespace Lykke.Job.OperationsCache.Services.InMemoryCache
+{
+    public class InMemoryStorage : IStorage
+    {
+        private readonly ConcurrentDictionary<string, CacheModel> _storage;
+
+        public InMemoryStorage()
+        {
+            _storage = new ConcurrentDictionary<string, CacheModel>();
+        }
+
+        public async Task<IEnumerable<HistoryEntry>> Get(string clientId)
+        {
+            if (_storage.TryGetValue(clientId, out CacheModel cachedValue))
+            {
+                return cachedValue.Records.Values;
+            }
+
+            return null;            
```

Comment:
I'd return empty collection here, but it's up to you.

Reply:
We have to use null